.github/workflows/publish_api_docs.yml

```diff
@@ -26,14 +26,10 @@ jobs:
           node-version: '14'
       - run: npm install
       - run: npm install html-minifier -g
-      - name: Setup environment
-        uses: ./.github/actions/setup_environment
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v5.0.0
         with:
-          cache_version: ${{ secrets.GH_ACTIONS_CACHE_KEY }}-api-docs
-          python-version: '3.8'
-          os: ubuntu-latest
-          # We need to install integrations in a different way to avoid conflicts
-          install_integrations: 'no'
+          python-version: '3.11'
```

Comment:
is there a reason that we use python 3.11?

Reply:
It is the fastest, any objections? No deps installed, so I pick fastest.

Reply:
Given how often we return to fix these things, I'm happy to keep it a future-facing version as it'll be a while (hopefully) before we return to update this functionality / code.

Reply:
no objection, was just surprised as I forgot that we already support 3.11